fenix/app/src/test/java/org/mozilla/fenix/components/metrics/DefaultMetricsStorageTest.kt

```diff
@@ -382,6 +384,120 @@ class DefaultMetricsStorageTest {
         assertTrue(updateSlot.captured > 0)
     }
 
+    @Test
+    fun `GIVEN first week activated threshold reached THEN will be sent`() = runTest(dispatcher) {
+        val currentTime = System.currentTimeMillis()
+        installTime = currentTime - (dayMillis * 5)
+        every { settings.adjustUseCount.value } returns 3
+        every { settings.adjustEarlySearchCount.value } returns 1
+        every { settings.adjustUserActivatedSent } returns false
+
+        val result = storage.shouldTrack(Event.GrowthData.UserActivated(fromSearch = false))
+
+        assertTrue(result)
+    }
+
+    @Test
+    fun `GIVEN first week activated threshold not reached THEN will not be sent`() = runTest(dispatcher) {
```

Comment:
I think we should update this test name to make it a little more clear that the "days of use" condition is the reason that the event isn't being sent. 

Additionally, I think we should add one more test where the days of use condition is met, but the search condition prevents the event from being sent.